radix-engine-interface/src/hooks.rs

```diff
@@ -16,8 +17,21 @@ pub struct OnDropInput {}
 
 pub type OnDropOutput = ();
 
+// TODO: expose generic information, but fully-detailed actor?
```

Comment:
Can we also add a comment to look to remove some of the too-specific fields (like `is_to_barrier`)

Reply:
Yes